src/bundle/Resources/views/themes/admin/ui/layout.html.twig

```diff
@@ -62,6 +62,7 @@
             window.ibexa.addConfig('richText', {{ ibexa_richtext_config|json_encode|raw }});
             window.ibexa.addConfig('errors', {
                 emailRegexp: /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/,
+                urlRegexp: /^(https?:\/\/|ftp:\/\/|www\.|\/)[^\s]+$/,
```

Comment:
```suggestion
                urlRegexp: /^(https?:\/\/|ftps?:\/\/|sftp:\/\/|www\.|\/)[^\s]+$/,
```
Also, maybe a problem, maybe not: `example.com` will be rejected, since it doesn't start with http/s or www.

Reply:
ezcontent/ezlocation urls are not supposed to be accepted here, right? I don't remember :)

Reply:
> ezcontent/ezlocation urls are not supposed to be accepted here, right? I don't remember :)

Updated regular expression for `ezlocation://`